core-bundle/tests/Contao/ControllerTest.php

```diff
@@ -348,4 +349,94 @@ public function testAddImageToTemplateLogsAndCreatesFallbackDataWhenNoFigureIsBu
 
         $this->assertSame($expectedTemplateData, get_object_vars($template));
     }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testAddToUrlWithoutQueryString(): void
+    {
+        define('TL_SCRIPT', '');
```

Comment:
If we're already working on this method we could also get rid of this deprecated constant, no?

Reply:
That is also not related to this PR. Maybe create a separate PR and fix it everywhere instead only in one place?

Reply:
I have added it to the Contao 5 project ideas.